### PR TITLE
add Nextcloud

### PR DESCRIPTION
**Pull request by jancborchardt**
This adds [Nextcloud](https://nextcloud.com), a completely open source fork of Owncloud.

Please review @nylira @alerque @strugee :)

### CODE REVIEW

**Comment by jancborchardt** [comment]
@strugee thanks for the review, fixed both! :) The CONTRIBUTING.md was a bit confusing which led me to edit all the other language files too – fixed that sentence in another pull request at https://github.com/nylira/prism-break/pull/1704

File: source/db/en-projects.json [hunk 1]
```diff
@@ -3918,6 +3918,117 @@
...
     "development_stage": "released",
+    "description": "A safe home for all your data",
+    "license_url": "https://github.com/nextcloud/server/blob/master/COPYING-README",
+    "logo": "nextcloud.png",
+    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.",
+    "privacy_url": "",
+    "source_url": "https://github.com/nextcloud",
+    "name": "Nextcloud",
+    "tos_url": "",
+    "url": "http://nextcloud.com",
```
Comment (strugee) [inline source/db/en-projects.json]: Nit: `https://`

File: source/db/en-projects.json [hunk 2]
```diff
@@ -3918,6 +3918,117 @@
...
+    "wikipedia_url": "https://en.wikipedia.org/wiki/Nextcloud",
+    "protocols": [
+      "CalDAV",
+      "CardDAV",
+      "IMAP",
```
Comment (strugee) [inline source/db/en-projects.json]: We don't currently have an `IMAP` protocol. Possibly we should, but in the meantime if you can get rid of this line that'd be great :)

File: source/db/en-projects.json [hunk 3]
```diff
@@ -3918,6 +3918,117 @@
...
+        "subcategories": [
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "Servers",
```
Comment (strugee) [inline source/db/en-projects.json]: Probably all the GNU/Linux, BSD, etc. entries should be removed. This is consistent with other selfhosted server stuff we list (Piwik, Prosody, Postfix)

File: source/db/en-projects.json (hunk 2 above)
Comment (jancborchardt) [inline source/db/en-projects.json]: Fixed – woud be good to add IMAP then ;)

File: source/db/en-projects.json (hunk 1 above)
Comment (jancborchardt) [inline source/db/en-projects.json]: Of course, thanks, fixed! :)

File: source/db/en-projects.json (hunk 3 above)
Comment (jancborchardt) [inline source/db/en-projects.json]: These refer to the desktop capabilities. Much like we have apps for iOS & Android and desktop clients for Windows and macOS, they also work on Linux. :) Piwik etc does not have desktop apps, so they don’t have entries there. But we sync files, and are deeply integrated via Calendar / Contacts sync, especially in GNOME for example.

Comment (strugee) [inline source/db/en-projects.json]: Ah, right. Forgot about that :)

**Pull request merged**